ipn/localapi/localapi.go

```diff
@@ -930,6 +939,18 @@ func (h *Handler) serveServeConfig(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func shouldPermitServeConfigUpdate(goos string, configIn *ipn.ServeConfig, h *Handler) bool {
+	if goos != "windows" {
+		return true
+	}
+
+	if !configIn.HasPathHandler() {
+		return true
+	}
+
+	return h.CallerIsLocalAdmin
```

Comment:
this could all be one line but I'm fine spelling it out in steps for clarity. Normally that's written like this because you want a nice place to put comments, but there are no comments here. Do you want some?

Or at least nuke the two empty lines? It's vertically tall enough?